### PR TITLE
Added vertical roles description

### PR DESCRIPTION
**Pull request by stankis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,14 @@ After that you can login to DataLens on http://localhost:8080 using the default
 
 You can use the same credentials to configure Zitadel and add new users using Zitadel control panel at http://localhost:8085/. **Don't forget to login there at least once to change the default password.**
 
+By default in DataLens with authentication every new user has datalens.viewer role which allows viewing collections and workbooks in read only mode. You are not allowed to modify or create any new entity with this role. In order to be able to edit or create new collection or workbook you need to have datalens.editor or datalens.admin role. To grant roles please open Zitadel at http://localhost:8085/ui/console/grants, then find user to whome you want to grant a new role, click on user and select new role.
+
+DataLens supports the following roles:
+
+datalens.viewer - allows view collections and workbooks of any user and doesn't allow to edit or create new ones.
+datalens.editor – includes datalens.viewer role and allows to edit, delete and create new collection or workbook for any user.
```
Comment (imsitnikov) [inline README.md:104]: `for any user` -> `in any place`?

File: README.md [hunk 2]
```diff
@@ -96,6 +96,14 @@ After that you can login to DataLens on http://localhost:8080 using the default
 
 You can use the same credentials to configure Zitadel and add new users using Zitadel control panel at http://localhost:8085/. **Don't forget to login there at least once to change the default password.**
 
+By default in DataLens with authentication every new user has datalens.viewer role which allows viewing collections and workbooks in read only mode. You are not allowed to modify or create any new entity with this role. In order to be able to edit or create new collection or workbook you need to have datalens.editor or datalens.admin role. To grant roles please open Zitadel at http://localhost:8085/ui/console/grants, then find user to whome you want to grant a new role, click on user and select new role.
```
Comment (imsitnikov) [inline README.md:99]: Does zitadel admin have `datalens.admin` be default?

File: README.md (hunk 1 above)
Comment (stankis) [reply README.md:104]: discussed offline, decided leave as it is

File: README.md (hunk 2 above)
Comment (stankis) [reply README.md:99]: I think it is a good point, will extend script with that

Comment (resure) [inline README.md:99]:
```suggestion
By default in DataLens with authentication enabled, all users have a datalens.viewer role. This allows them to use all collections and workbooks in read-only mode. They are not allowed to create or modify any objects with this role. To be able to create or edit objects, they need to have a datalens.editor or datalens.admin role. To grant these roles, open Zitadel at http://localhost:8085/ui/console/grants, then find the user to whom you want to grant a new role and click on the user and select the new role.
```

File: README.md [hunk 3]
```diff
@@ -96,6 +96,14 @@ After that you can login to DataLens on http://localhost:8080 using the default
...
+
+DataLens supports the following roles:
```
Comment (resure) [inline README.md:101]:
```suggestion
DataLens supports the following roles:

- `datalens.viewer`: allows viewing all collections and workbooks, but does not allow creating or editing any object.
- `datalens.editor`: includes the `datalens.viewer` role and allows creating, editing and deleting any object. 
- `datalens.admin`: currently equal to `datalens.editor`. In the future releases, users with this role will be able to manage system-wide settings and perform administrative functions.
```

**Review by imsitnikov — APPROVED** [review]

**Pull request merged**